index.html

```diff
@@ -205,6 +205,23 @@ <h1 class="about">About Myself</h1>
         </div>
     </section>
 
-    
+    <section class="contact-form">
+        <div class="form-1">
+            <img src="images/contact form shapes-mobile.svg" alt="" class="form-image">
+            <h1 class="Contact">Contact me</h1>
+            <p class="application">If you have an application you are interested 
+                in developing, a feature that you need built or a project that needs coding. 
+                I'd love to help with it </p> 
+
+            <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] Kindly create a _**Formspree**_ id from their [website](https://formspree.io/html/) and replace the value `form_id` with the actual _**Formspree**_ id as the value in the `action` attribute of the `form` element. 👍 💻